nri-config-generator/main.go

```diff
@@ -137,6 +138,14 @@ func generateOutput(exporterGenerator generator.Exporter, configGenerator genera
 	return output, err
 }
 
+func getExporterNameFromIntegration(integration string) string {
+	const (
+		integrationPrefix  = "nri-"
+		exporterNameFormat = "%s-exporter"
```

Comment:
usually we do not define the constant inside functions, but if we are sure we need them here only I guess it is fine

Reply:
That is when it's a global constant, in this case, since it's something that is only local to that function, I just added the constants inside the function (something really go style, because other langs don't let you do it) so it's easier to modify it, but it's the same as having it hard coded in the return.